OpenRA.Mods.Cnc/Projectiles/TeslaZap.cs

```diff
@@ -55,6 +56,12 @@ public TeslaZap(TeslaZapInfo info, ProjectileArgs args)
 			ticksUntilRemove = info.Duration;
 			damageDuration = info.DamageDuration > info.Duration ? info.Duration : info.DamageDuration;
 			target = args.PassiveTarget;
+
+			var pos = args.GuidedTarget.IsValidFor(args.SourceActor) ? args.GuidedTarget.CenterPosition : args.PassiveTarget;
+			zap = new TeslaZapRenderable(args.Source, 0, pos - args.Source,
+				info.Image, info.BrightSequence, info.BrightZaps, info.DimSequence, info.DimZaps, info.Palette);
+			var world = args.SourceActor.World;
+			world.ScreenMap.Add(this, pos, zap.ScreenBounds(null));
```

Comment:
Zaps jump about randomly, so I don't think we can trust the bounds from a single zap?  We would need to make a sufficiently larger box to cover all the places it could wander.